docs/api/action.md

```diff
@@ -48,7 +48,7 @@ Create a new action.
 
 *  **`response_handle`** nullable string, and must be a valid json-query, something like '.item.title'
 
-*  **`template`** nullable map where {:method -> <enum of GET, POST, PUT, DELETE, PATCH>, :url -> <string with length 1 or longer >, :body (optional) -> <nullable string>, :headers (optional) -> <nullable string>, :parameters (optional) -> <nullable sequence of map>, :parameter_mappings (optional) -> <nullable map>} with no other keys
+*  **`template`** nullable map where {:method -> <enum of GET, POST, PUT, DELETE, PATCH>, :url -> <string with length <= 1>, :body (optional) -> <nullable string>, :headers (optional) -> <nullable string>, :parameters (optional) -> <nullable sequence of map>, :parameter_mappings (optional) -> <nullable map>} with no other keys
```

Comment:
Shouldn't this be `>=1` ?

Reply:
Good catch. These docs are generated from source code, so we'll have to make the change upstream.